test/unit/client.integration.test.ts

```diff
@@ -31,6 +34,11 @@ function disconnectSdk (sdk: GenesysCloudWebrtcSdk): Promise<any> {
 }
 
 describe('Client (integration)', () => {
+  Object.defineProperty(window.navigator, 'hid', { get: () => ({
+    getDevices: () => { return [] }
+  })});
+  Object.defineProperty(window.navigator, 'locks', { get: () => ({})});
+  (window as any).BroadcastChannel = BroadcastChannel;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9175995)
```suggestion
  (window as unknown).BroadcastChannel = BroadcastChannel;
```